backend/bookchat/views.py

```
@@ -10,6 +10,9 @@
 from bookchat.models import User  #
 from df_chat.drf.viewsets import RoomViewSet
 from rest_framework.authentication import TokenAuthentication
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)